src/character.h

```diff
@@ -353,6 +353,7 @@ class Character : public Creature, public visitable<Character>
          */
         int get_item_position( const item *it ) const;
 
+        long int i_add_to_container(const item &it, const bool unloading);
```

Comment:
Probably needs a comment saying *what* the returned value indicates. Is it the amount that was added or the remaining amount? What does `unloading` indicate?